content/product/user_journey_maps.md

```diff
@@ -0,0 +1,21 @@
+# User journey mapping
+
+## About
+
+Sourcegraph's approach to user journey mapping is decentralized and just-in-time. In practice this means that no one team owns a central user journey map. Instead, maps are created as needed and in support of an outcome (as opposed to creating one as an artifact to just have around) because user journey maps:
+
+- differ based on the persona, level of detail, start and end, and ultimately the specific needs and goals of projects utilizing them.
+- start to slowly become out of date the second they are finished.
+- only takes 1-2 hours for an "MVP" map, and the process for creating them can be as valuable as the map itself.
+
+## Creating a map
+
+You're ready to create a user journey map when you've defined the following:
+
+- The outcome this map is in support of:
+- Persona (e.g. enterprise admin/dev tools team):
```

Comment:
@ebrodymoore - Are you able to use/link to the newly defined [personas](https://docs.google.com/presentation/d/1aQhcWoWd_LJXdAgEn7JBGnZV5pfN6UJyct2VV-ZiTXI/edit#slide=id.ge9365b5bc5_0_1246) for this?